Apps/Immunization/src/Models/ImmunizationEvent.cs

```diff
@@ -59,14 +59,14 @@ public class ImmunizationEvent
         /// Gets or sets the Immunization forecast.
         /// </summary>
         [JsonPropertyName("forecast")]
-        public ImmunizationForecast Forecast { get; set; } = new ImmunizationForecast();
+        public ImmunizationForecast? Forecast { get; set; } = null;
```

Comment:
We don't allow nullable in our code right? if so, can we init it as we usually do

Reply:
We allow nullable, just have to be explicitly defined like so